src/snarks/groth16/powersoftau_utils.hpp

```diff
@@ -0,0 +1,83 @@
+#ifndef __ZETH_SNARKS_GROTH_POWERSOFTAU_UTILS_HPP__
+#define __ZETH_SNARKS_GROTH_POWERSOFTAU_UTILS_HPP__
+
+#include "include_libsnark.hpp"
+
+#include <istream>
+
+namespace libzeth
+{
+
+using ppT = libff::default_ec_pp;
+
+/// Output from the first phase of the MPC (powersoftau).  The
+/// structure matches that data exactly (no indication of degree,
+/// etc), so that it can be loaded directly.
+class powersoftau
```

Comment:
What about renaming this class as `powersoftau_srs(/result/output)`?

Reply:
Renamed as suggested.